dask/dataframe/io/tests/test_csv.py

```diff
@@ -211,6 +213,21 @@ def test_skiprows(dd_read, pd_read, files):
         assert_eq(df, expected_df, check_dtype=False)
 
 
+@pytest.mark.parametrize('dd_read,pd_read,files,units',
+                         [(dd.read_csv, pd.read_csv, csv_files, csv_units_row),
+                          (dd.read_table, pd.read_table, tsv_files, tsv_units_row)])
+def test_skiprows_as_list(dd_read, pd_read, files, units):
+    files = {name: (comment_header + b'\n' +
+                    content.replace(b'\n', b'\n' + units, 1)) for name, content in files.items()}
+    n_comment_lines = len(comment_header.splitlines())
+    skip = list(range( n_comment_lines))
+    skip.append(n_comment_lines + 1)
```

Comment:
Is it possible to just hardcode this value? I think it'd make reading the test a bit easier. Would it be `skip = [0, 1, 2, 3, 5]`? My math may be off.